### ISSUE TITLE
'atomic verify' can return fatal error with local images

### ISSUE THREAD

**Issue by miabbott** [projectatomic/atomic#331]
After building an image which included a "Name", "Version", and "Release" LABEL, a subsequent use of `atomic verify` can return a fatal error.

```
# cat test_nvr/Dockerfile 
FROM registry.access.redhat.com/rhel7
LABEL Name="rhel7/test"
LABEL Version="9000"
LABEL Release="1"
CMD ["/bin/bash"]

# docker build -t rhel7_test_nvr test_nvr/
Sending build context to Docker daemon 2.048 kB
Step 1 : FROM registry.access.redhat.com/rhel7
 ---> bf63a676257a
Step 2 : LABEL Name "rhel7/test"
 ---> Running in 3720c117f170
 ---> d5073136cd89
Removing intermediate container 3720c117f170
Step 3 : LABEL Version "9000"
 ---> Running in 3a3e075ed21a
 ---> 9c2c8f3e3212
Removing intermediate container 3a3e075ed21a
Step 4 : LABEL Release "1"
 ---> Running in ae510a3eae47
 ---> 53eabd26e61b
Removing intermediate container ae510a3eae47
Step 5 : CMD /bin/bash
 ---> Running in fc1f2fdf0ee5
 ---> 262084d9ae71
Removing intermediate container fc1f2fdf0ee5
Successfully built 262084d9ae71

# docker images | grep nvr
rhel7_test_nvr                                               latest              262084d9ae71        5 seconds ago       203.2 MB

# atomic verify -v rhel7_test_nvr
time="2016-03-30T20:01:22Z" level=fatal msg="Please use a fully qualified repository name" 
```

This was seen using `atomic-1.9-4.gitff44c6a.el7.x86_64` and `docker-1.9.1-25.el7.x86_64` on RHELAH

**Comment by baude** [comment]
@rhatdan ill look at this soon...

**Comment by rhatdan** [comment]
Fixed in atomic-1.13
